🗄 Create Server README.md (#68)
loolabs/waterpark: server/README.md

# Waterpark Server

Waterpark's backend API server

## Getting Started

These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.

### Prerequisites

1. [Docker](https://www.docker.com/products/docker-desktop)
2. `.env` in the root directory of the monorepo

```
SERVER_PORT=3001
DB_DATABASE_NAME=clubs
DB_USERNAME=loolabs
DB_PASSWORD=loolabs
DB_PORT=5432
DATABASE_URL=postgresql://loolabs:loolabs@db/clubs
```

### Installing

1. Clone the mono repo.
1. Navigate to the server directory with `cd server`.

> 1. Run `docker-compose up` to start the express and postgres servers.

Author: You're right, great catch.

> 1. Clone the mono repo.
> 1. Navigate to the server directory with `cd server`.
> 1. Run `docker-compose up` to start the express and postgres servers.

Reviewer: ```suggestion
1. Clone the `waterpark` repository.
1. In the root directory of `waterpark`, run `docker-compose up` to start the express and postgres servers.
```

Author: Woops, thanks.

You should see messages in your terminal from the `server` and `postgres` containers, and finally be greeted with:

> ![](https://i.imgur.com/WhvhzFs.png)

Author: Smart man, TIL you can have local images in markdown. I moved all images to our repo.
